### PR TITLE
Update Simplified Chinese localization

### PR DESCRIPTION
**Pull request by Jerry23011**
Update Notifications Filter translations

@nixzhu Anytime 😄

### CODE REVIEW

File: IceCubesApp/Resources/Localization/Localizable.xcstrings [hunk 1]
```diff
@@ -34825,8 +34825,8 @@
         },
         "zh-Hans" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "People not following you"
+            "state" : "translated",
+            "value" : "没有关注你的人"
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/Localizable.xcstrings:34829]: 未关注你的人

File: IceCubesApp/Resources/Localization/Localizable.xcstrings [hunk 2]
```diff
@@ -34944,8 +34944,8 @@
         },
         "zh-Hans" : {
           "stringUnit" : {
-            "state" : "needs_review",
-            "value" : "People you don't follow"
+            "state" : "translated",
+            "value" : "你没有关注的人"
```
Comment (nixzhu) [inline IceCubesApp/Resources/Localization/Localizable.xcstrings:34948]: 你未关注的人

**Comment by Jerry23011** [comment]
They're fixed, thank you :)

**Review by nixzhu — APPROVED** [review]
LGTM

**Pull request merged**